lib/traject/psulib_config.rb

```diff
@@ -208,8 +206,28 @@
 to_field 'author_meeting_display_ssm', extract_marc('111abcdfgklnpqj'), trim_punctuation
 to_field 'addl_author_display_ssm', extract_marc('700aqbcdjk:710abcdfgjkln:711abcdfgjklnpq'), trim_punctuation
 
-# HathiTrust fields
-to_field 'hathitrust_struct', extract_hathi_data
+# HathiTrust access
+to_field 'ht_access_ss' do |_record, accumulator, context|
+  catkey = context.output_hash['id'].first
+
+  # Is the catkey in the Hathi overlap report? If so, write down allow or deny.
+  # relevant is an array of finds (often just one)
+  relevant = HATHI_OVERLAP_REPORT.find_all { |hr| hr['local_id'] == catkey  }
+
+  accumulator << if relevant.any?
+    # When there's more than one
+    if relevant.count > 1
+      access_array = relevant.collect {|hr| hr['access']}
+      if access_array.include? 'deny'
```

Comment:
I don't know that this is right but I think it is. Basically if there are dupe items in the overlap then I'm saying "deny" wins. 🤔 Should it be the other way around? Does mono versus multi in this case make a difference?

Reply:
in cases like 

```
2384942	1099502	mono	deny	ic
2384942	1099502	mono	allow	pd
2384942	1099502	multi	deny	ic
2384942	1099502	multi	allow	pd
```

`allow` wins (as per slack). I'll update the wiki.